docs/uptime-guide/security.asciidoc

```diff
@@ -52,7 +41,8 @@ PUT /_security/role/uptime
 [float]
 === Assign the role to a user
 
-Next, you'll need to create a user with both the `kibana_user`, and `uptime` roles. 
+Next, you'll need to create a user with both the `uptime` role, and another role with sufficient <<kibana-privileges, Kibana privileges>>,
```

Comment:
This link didn't build for me when running `docbldup` -- what's the best way for us to link to this?

Reply:
@legrego, I built the docs locally, and the links work fine